pybot/module_camera/UserCardsTracker.py

```diff
@@ -3,24 +3,26 @@
     card_contours_transform,
     rotate_top_left_corner_low_density_transform,
 )
-from module_webapp.dao import user
-from module_webapp.app import db
+from ..module_webapp.dao import user
+from ..module_webapp.app import db
 import cv2
 import os
 from cv2.typing import MatLike
 from typing import List, Tuple
 
 from sqlalchemy_media import StoreManager
-from module_camera import ImageComparator
+from .compare_images import ImageComparator
 from flask import Flask
 
-from module_webapp.models.user import UserResponse
+from ..module_webapp.models.user import UserResponse
+
+import pygame as pg
 
 
 class UserCardsTracker:
     def __init__(self, app: Flask):
         self.users, user_image_paths = self._get_users_info(app)
-        self.image_comparator = ImageComparator(user_image_paths)
+        self.image_comparator = ImageComparator(user_image_paths, sim_min_thresholds=(0.75, 0.85))
```

Comment:
move this from the constructor to the draw method directly.

The draw method should take 2 optional parameters (thresholds).